trio/_threads.py

```diff
@@ -425,7 +427,16 @@ def from_thread_run_sync(fn, *args, trio_token=None):
     def callback(q, fn, args):
         @disable_ki_protection
         def unprotected_fn():
-            return fn(*args)
+            call = fn(*args)
+
+            if inspect.iscoroutine(call):
+                call.close()
```

Comment:
To avoid `RuntimeWarning`s

Reply:
Please say that in a comment in the code, so that people reading the code will understand it. Comments in the PR don't help anyone except the people reviewing the PR.